src/lib/utils/enhanceConfiguration.ts

```diff
@@ -16,7 +14,7 @@ function enhanceConfiguration(
   const partialConfiguration = widget.getConfiguration(configuration);
```

Comment:
The function `getConfiguration` expects a `PlainSearchParameters`, but I guess it's fine since both type are compatible.